io.github.ecotubehq.player.json

```diff
@@ -0,0 +1,360 @@
+{
+    "app-id": "io.github.ecotubehq.player",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "45",
+    "sdk": "org.gnome.Sdk",
+    "command": "ecotube",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=all",
```

Comment:
why is device all needed?

Reply:
So player can read files from all devices

Reply:
what do you mean "all devices"?

Reply:
this is not what this does.

Reply:
@hfiguiere I ca see from the documentation [https://docs.flatpak.org/en/latest/sandbox-permissions.html#device-access](here) that we can use `--device=all ` to access devices like generic USB drives, webcams, CD/DVD drives etc. Am I doing something wrong?